components/hal/include/hal/uart_types.h

```diff
@@ -114,19 +140,6 @@ typedef struct {
     uint8_t xoff_thrd;       /*!< If the software flow control is enabled and the data amount in rxfifo is more than xoff_thrd, an xoff_char will be sent*/
 } uart_sw_flowctrl_t;
 
-/**
- * @brief UART configuration parameters for uart_param_config function
- */
-typedef struct {
```

Comment:
Sounds like uart_types.h has changed a lot. What about others SoCs? Does it all support this new change?

Reply:
These changes should affect only C6, as they are related to LP system